components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
I added it here because the censusmember gets its data from the service rumor so I think we need this. but i could be missing something!